### ISSUE TITLE
Question: New architecture, bridgeless mode, ...

### ISSUE THREAD

**Issue by DriesH** [voize-gmbh/reakt-native-toolkit#45]
Hi,

First of all, super cool library and idea!

At my company, we already use kotlin multiplatform a lot for our native apps.
And we also see a huge benefit in using it when developing native modules for some of our react-native apps. But since react-native is moving closer and closer to finally completing the new architecture work, it would be nice to know if you guys are planning to also make this toolkit compatible with TurboModules and bridgeless mode?

Thanks!

**Comment by Legion2** [comment]
Hi, we are currently stuck with an old React Native version and also depend on some RN modules which do not support the new architecture. We tested the toolkit with old and new versions. Because the old module system still works with new React Native versions there is no incentive for us to migrate to the TurboModules and bridgeless mode.
In the future we are also planning to migrate to Compose Multiplatform instead of using React Native. But there is a long road to go.

TLDR; currently we are not planning to support TurboModules and bridgeless mode.

**Comment by DriesH** [comment]
Thanks for the response!
